evals/sampling_eval.py

```diff
@@ -0,0 +1,129 @@
+import torch
+import seaborn as sns
+import torch.nn as nn
+import matplotlib.pyplot as plt
+from omegaconf import OmegaConf
+from huggingface_hub.file_download import hf_hub_download
+
+from sample import Generator
+from train import preprocess_dataset
+from models.reverse_diffusion import Unet
+from models.forward_diffusion import ForwardDiffusion
+from models.velocity_time_encoder import VelocityTimeEncoder
+
+
+def eval_generation(
+    gen: Generator,
+    cfg: OmegaConf,
+    conditioning_model: nn.Module = None,
+    classifier_free_guidance_scale: float = 3.0,
+    batch_size: int = 2,
+):
+    _, loader, _ = preprocess_dataset("JasiekKaczmarczyk/giant-midi-sustain-quantized", batch_size, 1, overfit_single_batch=False)
+
+    batch = next(iter(loader))
+
+    velocity = batch["velocity"]
+
+    velocity_bin = batch["velocity_bin"].to(cfg.train.device)
+    dstart_bin = batch["dstart_bin"].to(cfg.train.device)
+    duration_bin = batch["duration_bin"].to(cfg.train.device)
+
+    if conditioning_model is not None:
+        with torch.no_grad():
+            label_emb = conditioning_model(velocity_bin, dstart_bin, duration_bin)
+    else:
+        label_emb = None
+
+    noise = torch.randn(velocity.size()).to(cfg.train.device)
+
+    # if intermediate_outputs=True returns dict of intermediate signals else only denoised signal
+    fake_velocity = gen.sample(
+        noise, label_emb=label_emb, intermediate_outputs=True, classifier_free_guidance_scale=classifier_free_guidance_scale
+    )
```

Comment:
```suggestion
    fake_velocity = gen.sample(
        x=noise,
        conditional_embedding=conditional_embedding,
        intermediate_outputs=True,
        classifier_free_guidance_scale=classifier_free_guidance_scale,
    )
```

Please use named parameters and break lines in case of multiple non-idiomatic arguments